src/Poster.js

```diff
@@ -2,11 +2,20 @@ import React from "react";
 import "./css/poster.css";
 
 
-const Poster = ({imagePath, title, id, showMovieSummary}) => {
+const Poster = ({imagePath, backdropPath, title, rating, id, showMovieSummary}) => {
     return (
-      <div  id={id} onClick={(event) => showMovieSummary(event)} className="poster">
-        <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
-        <h3 id={id}>{title}</h3>
+      <div id={id} onClick={(event) => showMovieSummary(event)} className="poster">
+       <div className="poster-inner">
+        <div className="poster-front">
+          <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
+          <h3 id={id}>{title}</h3>
+        </div>
+        <div className="poster-back">
```

Comment:
This was a really good idea to include a "back" to the cards to show a bit more.